weasyl/resetpassword.py

```diff
@@ -31,27 +31,30 @@ def request(form):
     if not user:
         raise WeasylError("loginRecordMissing")
 
-    # Check the user's email address against the provided e-mail address,
-    # raising an exception if there is a mismatch
+    # Check the user's email address against the provided e-mail address
```

Comment:
Following up on the parenthesized bit from the last comment: maybe this form shouldn’t ask for a username at all?